code/modules/implants/implantcase.dm

```diff
@@ -0,0 +1,103 @@
+/obj/item/weapon/implantcase
+	name = "Glass Case"
```

Comment:
not a proper noun

Reply:
you're not a proper noun
"kurfursten"

Reply:
Too bad for you that our repo is not in German, where all nouns are capitalized.

Reply:
*dabs in german